dawn/setup.py

```diff
@@ -70,30 +73,30 @@ def run(self):
         # Installing in editable/develop mode builds the extension in the original build path,
         # but a regular `pip install` copies the full tree to a temporary folder
         # before building, which makes CMake fail if a CMake cache had been already generated.
-        self.bundle_build_temp = str(os.path.join(BUNDLE_ABS_DIR, "build"))
-        cmake_cache_file = os.path.join(self.bundle_build_temp, "CMakeCache.txt")
+        # self.bundle_build_tmp = str(os.path.join(BUNDLE_ABS_DIR, "build"))
+        self.build_tmp = str(os.path.join(DAWN_ABS_DIR, "build"))
+        cmake_cache_file = os.path.join(self.build_tmp, "CMakeCache.txt")
         if os.path.exists(cmake_cache_file):
             with open(cmake_cache_file, "r") as f:
                 text = f.read()
-                m = re.search(r"\s*dawn_BINARY_DIR\s*:\s*STATIC\s*=\s*([\w/\\]+)\s*", text)
+                m = re.search(r"\s*Dawn_BINARY_DIR\s*:\s*STATIC\s*=\s*([\w/\\]+)\s*", text)
                 cache_build_dir = m.group(1) if m else ""
-                if str(self.bundle_build_temp) != cache_build_dir:
-                    shutil.rmtree(self.bundle_build_temp, ignore_errors=False)
-                    shutil.rmtree(os.path.join(BUNDLE_ABS_DIR, "install"), ignore_errors=True)
+                if str(self.build_tmp) != cache_build_dir:
+                    shutil.rmtree(self.build_tmp, ignore_errors=False)
+                    shutil.rmtree(os.path.join(DAWN_DIR, "install"), ignore_errors=True)
                     assert not os.path.exists(cmake_cache_file)
-        os.makedirs(self.bundle_build_temp, exist_ok=True)
+        os.makedirs(self.build_tmp, exist_ok=True)
+        os.makedirs(os.path.join(DAWN_DIR, "install"), exist_ok=True)
```

Comment:
Why does this use `DAWN_DIR` but `build_tmp` uses `DAWN_ABS_DIR`?

Reply:
oversight